brands/amenity/cafe.json

```diff
@@ -1005,6 +1099,87 @@
       "takeaway": "yes"
     }
   },
+  "amenity/cafe|貢茶 Gong Cha": {
+    "countryCodes": ["hk"],
```

Comment:
I just realized Nominatim considers Hong Kong and Macau to have a country code of `cn`, so an editor like iD would ignore this preset entirely. Yet the tagging would definitely differ between Hong Kong and mainland China due to the presence of English and also the differences between traditional and simplified Chinese. Would it be OK to put both entries under `cn`? Or should we begin distinguishing Hong Kong from China for brand purposes?